py/torch_tensorrt/dynamo/conversion/impl/select.py

```diff
@@ -127,11 +151,14 @@ def index(
         rank = len(input_shape)
         adv_indx_count = len(adv_indx_indices)
         dim_tensor_list = []
+        dim_list = []
 
         for i in range(rank):
             dim = input_shape[i]
             dim_tensor = get_trt_tensor(ctx, dim, name + f"_individual_dim_{i}")
-            # dim_tensor_list is a list of tensors
+            # dim_tensor_list is a list of tensors or numpy
+            if is_numpy:
+                dim_list.append(dim)
```

Comment:
Why is a different list needed here if we have the `is_numpy` flag?

Reply:
As in - could `dim_list` just hold the `ITensors` _or_ the `numpy` arrays, or do you need both? If both are needed, can you just use `input_shape` in place of `dim_list`, as they should be the same